### PR TITLE
Increased Coverage to Post Upgrade Existance tests

### PR DESCRIPTION
**Pull request by jyejare**
Added coverage for more entities listed and checked only in issue #552 .

### CODE REVIEW

File: upgrade_tests/test_existance_relations/test_contenthosts.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('content-host', 'name')
+)
+def test_positive_contenthosts_by_name(pre, post):
+    """Test all content hosts are existing after upgrade by names
+
+    :id: aa92463b-e693-4c30-b0cb-e2cafdab1c7f
+
+    :assert: All content hosts should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_contenthosts.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_contenthosts.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('content-host', 'installable errata')
+)
+def test_positive_installable_erratas_by_name(pre, post):
+    """Test all content hosts installable erratas are existing after upgrade
+
+    :id: bc40b921-c39b-4cd0-9816-87b53d1af352
+
+    :assert: All chosts installable erratas should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_contenthosts.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discovery.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery', 'name')
+)
+def test_positive_discovery_by_name(pre, post):
+    """Test all architectures are existing after upgrade by names
+
+    :id: 2322766f-0731-4e80-bf54-d48a8756406d
+
+    :assert: All architectures should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discovery.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discovery.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery', 'mac')
+)
+def test_positive_discovery_by_mac(pre, post):
+    """Test discovered hosts mac is retained after upgrade
+
+    :id: 348a11f1-e7c2-4ff5-b36c-c79626ff2142
+
+    :assert: All discovered hosts mac should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discovery.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discovery.py [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery', 'cpus')
+)
+def test_positive_discovery_by_cpus(pre, post):
+    """Test discovered hosts cpus are retained after upgrade
+
+    :id: 733663f6-4bee-4e0d-b4ed-35ac2e0e6370
+
+    :assert: All discovered hosts cpus should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discovery.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discovery.py [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery', 'memory')
+)
+def test_positive_discovery_by_memory(pre, post):
+    """Test discovered hosts memory allocation is retained after upgrade
+
+    :id: 91d2c395-d788-45c8-b722-051fbed18d38
+
+    :assert: All discovered hosts memory allocation should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discovery.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discovery.py [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery', 'disk count')
+)
+def test_positive_discovery_by_disc_counts(pre, post):
+    """Test discovered hosts disc counts are retained after upgrade
+
+    :id: ddb9c37c-4287-4419-b890-8a7891a333f0
+
+    :assert: All discovered hosts disk counts should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discovery.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discovery.py [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery', 'disks size')
+)
+def test_positive_discovery_by_disc_size(pre, post):
+    """Test discovered hosts disc size are retained after upgrade
+
+    :id: ad71e779-cded-4ba7-aaf2-ff0d138b3613
+
+    :assert: All discovered hosts disk size should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discovery.py]: s/assert/expectedresults

And indent the second line.

File: upgrade_tests/test_existance_relations/test_discovery.py [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery', 'subnet')
+)
+def test_positive_discovery_by_subnet(pre, post):
+    """Test discovered hosts subnet is retained after upgrade
+
+    :id: c5218155-95a4-4a90-b853-76f843bb07c0
+
+    :assert: All discovered hosts subnet should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discovery.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_discoveryrulels.py [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery_rule', 'name')
+)
+def test_positive_discovery_rules_by_name(pre, post):
+    """Test all discovery rules are existing after upgrade by name
+
+    :id: 0d7e8920-5717-4196-af8a-977cfba33184
+
+    :assert: All discovery rules should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discoveryrulels.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discoveryrulels.py [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery_rule', 'priority')
+)
+def test_positive_discovery_rules_by_priority(pre, post):
+    """Test all discovery rules priorities are existing after upgrade
+
+    :id: f2a1c6e6-d025-463c-a837-4f4657106f1e
+
+    :assert: All discovery rules priorities should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discoveryrulels.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discoveryrulels.py [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery_rule', 'search')
+)
+def test_positive_discovery_rules_by_search(pre, post):
+    """Test all discovery rules search are existing after upgrade
+
+    :id: ef1944c4-62f6-447e-90d9-f8ed95eb35de
+
+    :assert: All discovery rules search should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discoveryrulels.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discoveryrulels.py [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+    "pre,post",
+    compare_postupgrade('discovery_rule', 'host group')
+)
+def test_positive_discovery_rules_by_hostgroup(pre, post):
+    """Test all discovery rules hostgroup associations are existing after
+    upgrade
+
+    :id: da605ae6-cdf8-49f9-87f6-c1cdfc411f90
+
+    :assert: All discovery rules hostgroups should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discoveryrulels.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discoveryrulels.py [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('discovery_rule', 'hosts limit')
+)
+def test_positive_discovery_rules_by_hostslimit(pre, post):
+    """Test all discovery rules hosts limit are retained after upgrade
+
+    :id: a9c59324-85eb-4295-8f2d-6f2e783a63dd
+
+    :assert: All discovery rules hosts limit should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discoveryrulels.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_discoveryrulels.py [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+    "pre,post",
+    compare_postupgrade('discovery_rule', 'enabled')
+)
+def test_positive_discovery_rules_by_enablement(pre, post):
+    """Test all discovery rules enablement and disablement is existing after
+    upgrade
+
+    :id: 7b71be69-1c60-43e8-bbfb-938565ef8eee
+
+    :assert: All discovery rules enablement and disablement should be retained
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_discoveryrulels.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_filter.py [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+    "pre,post",
+    compare_postupgrade('filter', 'resource type')
+)
+def test_positive_filters_by_resource_type(pre, post):
+    """Test all filters of all roles are existing after upgrade by resource
+    types
+
+    :id: 362f4b0c-49bb-424d-92e4-446ec59b8c5c
+
+    :assert: All filters of all roles should be retained post upgrade by
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_filter.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_filter.py [hunk 17]
```diff
@@ -0,0 +1,79 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('filter', 'search')
+)
+def test_positive_filters_by_search(pre, post):
+    """Test all filters search criteria is existing after upgrade
+
+    :id: da2dd076-f0e6-45ee-8a5d-99f2e083aabc
+
+    :assert: All filters search criteria should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_filter.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_filter.py [hunk 18]
```diff
@@ -0,0 +1,79 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('filter', 'unlimited?')
+)
+def test_positive_filters_by_unlimited_check(pre, post):
+    """Test all filters unlimited criteria is existing after upgrade
+
+    :id: abf65640-dc9b-415e-846d-0df43391228f
+
+    :assert: All filters unlimited criteria should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_filter.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_filter.py [hunk 19]
```diff
@@ -0,0 +1,79 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('filter', 'role')
+)
+def test_positive_filters_by_role(pre, post):
+    """Test all filters association with role is existing after upgrade
+
+    :id: dffdc0ac-a4b5-4b70-8e4d-fb37765f75ed
+
+    :assert: All filters association with role should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_filter.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_filter.py [hunk 20]
```diff
@@ -0,0 +1,79 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('filter', 'permissions')
+)
+def test_positive_filters_by_permissions(pre, post):
+    """Test all filters all permissions are existing after upgrade
+
+    :id: dd4fab7e-bd8f-4645-8aab-42a14ffe3a0e
+
+    :assert: All filters all permissions should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_filter.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_gpg.py [hunk 21]
```diff
@@ -0,0 +1,20 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('gpg', 'name')
+)
+def test_positive_gpg_keys_by_name(pre, post):
+    """Test all gpg keys are existing after upgrade by names
+
+    :id: 23b96c3e-2510-4886-91e6-9864f0d5e3e5
+
+    :assert: All gpg keys should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_gpg.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_lifecycleenvs.py [hunk 22]
```diff
@@ -0,0 +1,21 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('lifecycle-environment', 'name')
+)
+def test_positive_lifecycle_envs_by_name(pre, post):
+    """Test all lifecycle envs are existing after upgrade by names
+
+    :id: 4bb9c13a-b573-4f03-b2b3-65592e275eb1
+
+    :assert: All lifecycle envs should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_lifecycleenvs.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_mediums.py [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('medium', 'name')
+)
+def test_positive_mediums_by_name(pre, post):
+    """Test all OS mediums are existing after upgrade by names
+
+    :id: fe7786be-61ea-4276-81c7-6228389c4b84
+
+    :assert: All OS mediums should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_mediums.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_project.py [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('product', 'name')
+)
+def test_positive_products_by_name(pre, post):
+    """Test all products are existing after upgrade by names
+
+    :id: 3dea1ee4-ed57-4341-957a-d9b1813ff4db
+
+    :assert: All products should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_project.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_project.py [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+    "pre,post",
+    compare_postupgrade('product', 'repositories')
+)
+def test_positive_products_by_repositories(pre, post):
+    """Test all products association with their repositories are existing after
+    upgrade
+
+    :id: cb3b838b-d69d-4de9-9ebb-bbc6143ecdbf
+
+    :assert: Repositories of all products should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_project.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_ptables.py [hunk 26]
```diff
@@ -0,0 +1,21 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('partition-table', 'name')
+)
+def test_positive_partition_tables_by_name(pre, post):
+    """Test all partition tables are existing after upgrade by names
+
+    :id: 7832ab52-75e5-4451-aee3-5b208ced0e67
+
+    :assert: All architectures should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_ptables.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_puppet_module_class.py [hunk 27]
```diff
@@ -0,0 +1,35 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('puppet-class', 'name')
+)
+def test_positive_puppet_classes_by_name(pre, post):
+    """Test all puppet classes are existing after upgrade by names
+
+    :id: 44e7617c-4092-42bd-9b28-907c034966f7
+
+    :assert: All puppet classes should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_puppet_module_class.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_puppet_module_class.py [hunk 28]
```diff
@@ -0,0 +1,35 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('puppet-module', 'name')
+)
+def test_positive_puppet_modules_by_name(pre, post):
+    """Test all puppet modules are existing after upgrade by names
+
+    :id: 9749e8e0-8424-4825-9350-5475413eba8e
+
+    :assert: All puppet modules should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_puppet_module_class.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_puppetenvironment.py [hunk 29]
```diff
@@ -0,0 +1,21 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('environment', 'name')
+)
+def test_positive_puppet_envs_by_name(pre, post):
+    """Test all puppet envs are existing after upgrade by names
+
+    :id: d144094d-48fd-450b-8570-bd53758e090f
+
+    :assert: All puppet envs should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_puppetenvironment.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_repository.py [hunk 30]
```diff
@@ -0,0 +1,51 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('repository', 'name')
+)
+def test_positive_repositories_by_name(pre, post):
+    """Test all repositories are existing after upgrade by names
+
+    :id: 13811137-89f7-4dc7-b4b5-4aed91546bd5
+
+    :assert: All repositories should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_repository.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_repository.py [hunk 31]
```diff
@@ -0,0 +1,51 @@
...
+    "pre,post",
+    compare_postupgrade('repository', 'product')
+)
+def test_positive_repositories_by_product(pre, post):
+    """Test all repositories association with products are existing after
+    upgrade
+
+    :id: 24130f2e-4eef-4038-8ae6-14613c79e34a
+
+    :assert: All repositories association to its product should be retained
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_repository.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_repository.py [hunk 32]
```diff
@@ -0,0 +1,51 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('repository', 'content type')
+)
+def test_positive_repositories_by_url(pre, post):
+    """Test all repositories urls are existing after upgrade
+
+    :id: 0776a63f-863e-481d-a7a4-87e449029914
+
+    :assert: All repositories urls should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_repository.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_scparams.py [hunk 33]
```diff
@@ -0,0 +1,67 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('sc-param', 'parameter')
+)
+def test_positive_smart_params_by_name(pre, post):
+    """Test all smart parameters are existing after upgrade by names
+
+    :id: 44113fb7-eab2-439b-986c-6110a1c15d54
+
+    :assert: All smart parameters should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_scparams.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_scparams.py [hunk 34]
```diff
@@ -0,0 +1,67 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('sc-param', 'default value')
+)
+def test_positive_smart_params_by_default_value(pre, post):
+    """Test all smart parameters default values are retained after upgrade
+
+    :id: 35a94fb5-5601-4b85-b23a-dd3ccb945bd6
+
+    :assert: All smart parameters default values should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_scparams.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_scparams.py [hunk 35]
```diff
@@ -0,0 +1,67 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('sc-param', 'override')
+)
+def test_positive_smart_params_by_override(pre, post):
+    """Test all smart parameters override check is retained after upgrade
+
+    :id: 9f045338-8a79-43b1-a22c-45e79e8dbc56
+
+    :assert: All smart parameters override check should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_scparams.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_scparams.py [hunk 36]
```diff
@@ -0,0 +1,67 @@
...
+    "pre,post",
+    compare_postupgrade('sc-param', 'puppet class')
+)
+def test_positive_smart_params_by_puppet_class(pre, post):
+    """Test all smart parameters associations with its puppet class is retained
+    after upgrade
+
+    :id: 86714406-afcf-45a8-8db9-07ea03251cfa
+
+    :assert: All smart parameters associations with puppet classes should be
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_scparams.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_smartvariables.py [hunk 37]
```diff
@@ -0,0 +1,67 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('smart-variable', 'name')
+)
+def test_positive_smart_variables_by_name(pre, post):
+    """Test all smart variables are existing after upgrade by names
+
+    :id: d2543c28-135d-4e8f-8fe6-f510f74f51b9
+
+    :assert: All smart variables should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_smartvariables.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_smartvariables.py [hunk 38]
```diff
@@ -0,0 +1,67 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('smart-variable', 'default value')
+)
+def test_positive_smart_variables_by_default_value(pre, post):
+    """Test all smart variables default values are retained after upgrade
+
+    :id: c8337fbc-9c26-408a-a6ac-6f4886aabcdf
+
+    :assert: All smart variables default values should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_smartvariables.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_smartvariables.py [hunk 39]
```diff
@@ -0,0 +1,67 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('smart-variable', 'type')
+)
+def test_positive_smart_variables_by_type(pre, post):
+    """Test all smart variables override check is retained after upgrade
+
+    :id: 401e491c-bb54-4d2e-88a7-b6b6a9c033e3
+
+    :assert: All smart variables override check should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_smartvariables.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_smartvariables.py [hunk 40]
```diff
@@ -0,0 +1,67 @@
...
+    "pre,post",
+    compare_postupgrade('smart-variable', 'puppet class')
+)
+def test_positive_smart_variables_by_puppet_class(pre, post):
+    """Test all smart variables associations with its puppet class is retained
+    after upgrade
+
+    :id: 97721211-4cfe-4170-8e9b-5dd622e0ae81
+
+    :assert: All smart variables associations with puppet classes should be
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_smartvariables.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_subscriptions.py [hunk 41]
```diff
@@ -0,0 +1,91 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('subscription', 'name')
+)
+def test_positive_subscriptions_by_name(pre, post):
+    """Test all subscriptions are existing after upgrade by names
+
+    :id: 535d6529-27cb-4c6f-959e-6d0684e77aa6
+
+    :assert: All subscriptions should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_subscriptions.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_subscriptions.py [hunk 42]
```diff
@@ -0,0 +1,91 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('subscription', 'uuid')
+)
+def test_positive_subscriptions_by_uuid(pre, post):
+    """Test all subscriptions uuids are existing after upgrade
+
+    :id: 535d6529-27cb-4c6f-959e-6d0684e77aa6
+
+    :assert: All subscriptions uuids should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_subscriptions.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_subscriptions.py [hunk 43]
```diff
@@ -0,0 +1,91 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('subscription', 'support')
+)
+def test_positive_subscriptions_by_support(pre, post):
+    """Test all subscriptions support status is retained after upgrade
+
+    :id: 535d6529-27cb-4c6f-959e-6d0684e77aa6
+
+    :assert: All subscriptions support status should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_subscriptions.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_subscriptions.py [hunk 44]
```diff
@@ -0,0 +1,91 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('subscription', 'quantity')
+)
+def test_positive_subscriptions_by_quantity(pre, post):
+    """Test all subscriptions quantities are retained after upgrade
+
+    :id: 535d6529-27cb-4c6f-959e-6d0684e77aa6
+
+    :assert: All subscriptions quantities should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_subscriptions.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_subscriptions.py [hunk 45]
```diff
@@ -0,0 +1,91 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('subscription', 'consumed')
+)
+def test_positive_subscriptions_by_consumed(pre, post):
+    """Test all subscriptions consumed status is retained after upgrade
+
+    :id: 535d6529-27cb-4c6f-959e-6d0684e77aa6
+
+    :assert: All subscriptions consumed status should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_subscriptions.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_subscriptions.py [hunk 46]
```diff
@@ -0,0 +1,91 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('subscription', 'end date')
+)
+def test_positive_subscriptions_by_end_date(pre, post):
+    """Test all subscriptions end date status is retained after upgrade
+
+    :id: 535d6529-27cb-4c6f-959e-6d0684e77aa6
+
+    :assert: All subscriptions end date status should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_subscriptions.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_syncplan.py [hunk 47]
```diff
@@ -0,0 +1,64 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('sync-plan', 'name')
+)
+def test_positive_syncplans_by_name(pre, post):
+    """Test all sync plans are existing after upgrade by names
+
+    :id: 8030bff2-455e-4b1a-8b62-0596465ef2da
+
+    :assert: All sync plans should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_syncplan.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_syncplan.py [hunk 48]
```diff
@@ -0,0 +1,64 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('sync-plan', 'start date')
+)
+def test_positive_syncplans_by_start_date(pre, post):
+    """Test all sync plans start date is retained after upgrade
+
+    :id: 8106ddf2-701c-4c58-8246-b0122195fa5d
+
+    :assert: All sync plans start date should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_syncplan.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_syncplan.py [hunk 49]
```diff
@@ -0,0 +1,64 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('sync-plan', 'interval')
+)
+def test_positive_syncplans_by_interval(pre, post):
+    """Test all sync plans interval time is retained after upgrade
+
+    :id: 058eeba9-9a4d-44c5-a759-48c3199b70f0
+
+    :assert: All sync plans interval time should be retained post upgrade
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_syncplan.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_syncplan.py [hunk 50]
```diff
@@ -0,0 +1,64 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('sync-plan', 'enabled')
+)
+def test_positive_syncplans_by_enablement(pre, post):
+    """Test all sync plans enablement and disablement is retained after upgrade
+
+    :id: a90e8c93-74b5-49f8-9c08-4fba7903635c
+
+    :assert: All sync plans enablement and disablement should be retained post
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_syncplan.py]: s/assert/expectedresults

And indent the second line

File: upgrade_tests/test_existance_relations/test_template.py [hunk 51]
```diff
@@ -0,0 +1,21 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('template', 'name')
+)
+def test_positive_templates_by_name(pre, post):
+    """Test all templates are existing after upgrade by names
+
+    :id: fce33637-8e7b-4ccf-a9fb-47f0e0607f83
+
+    :assert: All templates should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_template.py]: s/assert/expectedresults

File: upgrade_tests/test_existance_relations/test_usergroup.py [hunk 52]
```diff
@@ -0,0 +1,21 @@
...
+@pytest.mark.parametrize(
+    "pre,post",
+    compare_postupgrade('user-group', 'name')
+)
+def test_positive_usergroups_by_name(pre, post):
+    """Test all usergroups are existing after upgrade by names
+
+    :id: 62e8bbca-25f5-403c-b868-7f0bc11ff341
+
+    :assert: All user groups should be retained post upgrade by names
```
Comment (elyezer) [inline upgrade_tests/test_existance_relations/test_usergroup.py]: s/assert/expectedresults

**Comment by jyejare** [comment]
@elyezer : I worked on your comments by replacing 'assert' with 'expectedresults' in docstrings.

Please review again and merge.

**Pull request merged**